sdk/client/src/listeners/first-load-listeners.tsx

```diff
@@ -155,10 +155,15 @@ export class FirstLoadListeners {
 		sThis: FirstLoadListeners,
 		options: HighlightClassOptions,
 	): void {
-		sThis._backendUrl =
+		const _backendUrl =
 			options?.backendUrl ||
 			import.meta.env.REACT_APP_PUBLIC_GRAPH_URI ||
 			'https://pub.highlight.run'
+		const otlpEndpoint =
+			options.otlpEndpoint ?? 'https://otel.highlight.io:4318'
```

Comment:
@Vadman97 is this correct to include the port on this URL, or can/should we leave it off?

Reply:
hmm probably better to leave it off, since https://otel.highlight.io (port 443) is also valid